python/nav/bin/update_ouis.py

```diff
@@ -30,10 +30,12 @@
 
 from nav.models.oui import OUI
 from nav.macaddress import MacPrefix
+from nav.logs import init_generic_logging
 
 _logger = logging.getLogger(__name__)
 
 FILE_URL = "https://standards-oui.ieee.org/oui/oui.txt"
+LOGFILE = "update_ouis.log"
```

Comment:
I think should be low-profile enough that it doesn't need its whole own log file.  I think a good strategy would be log directly to stderr, and use only debug-level logging.  If run as a cron job, any errors would become an e-mail to the admin, while successful runs stay quiet and out-of-the-way.